include/vcpkg/registries.h

```diff
@@ -162,8 +164,8 @@ namespace vcpkg
                                                                      Path path,
                                                                      std::string baseline);
 
-    ExpectedL<std::vector<std::pair<SchemedVersion, std::string>>> get_builtin_versions(const VcpkgPaths& paths,
-                                                                                        StringView port_name);
+    ExpectedL<Optional<std::vector<std::pair<SchemedVersion, std::string>>>> get_builtin_versions(
```

Comment:
With these return types I'm grateful that `auto` exists now.

Reply:
I thought the same thing when I looked at this lol